viewer/packages/cad-geometry-loaders/src/CadModelUpdateHandler.ts

```diff
@@ -110,10 +106,7 @@ export class CadModelUpdateHandler {
       map(createDetermineSectorsInput), // Map from array to interface (enables destructuring)
       filter(loadingEnabled), // should we load?
       mergeMap(async x => loadSectors(x)),
-      mergeMap(x => x),
-      finalize(() => {
-        this._sectorRepository.clear(); // clear the cache once this is unsubscribed from.
-      })
```

Comment:
What happens in practice here is that the cache is cleared when a model unsubscribes. I don't nessesarly believe this is correct, because of you have N models and remove one, then entire cache is cleared. But will add a cache clear on remove model which will have the same effect, then we can decide if we want to redesign this later.